pwiz_tools/Skyline/Test/FeatureTooltipTest.cs

```diff
@@ -41,8 +41,8 @@ public void TestFeatureTooltips()
                 var tooltip = calculator.Tooltip;
                 if (string.IsNullOrEmpty(tooltip))
                 {
-                    Console.Out.WriteLine("<data name=\"{0}\" xml:space=\"preserve\"><value></value></data>",
-                        calculator.FullyQualifiedName);
+                    // Console.Out.WriteLine("<data name=\"{0}\" xml:space=\"preserve\"><value></value></data>",
```

Comment:
This line of code only executes once it has been determined that the test is going to fail, and its job is to output text that a developer could copy and paste into "FeatureTooltips.resx".